### DISCUSSION TITLE
React Query Devtools

### DISCUSSION THREAD

**Discussion by un0211** [Part3-Team15/taskify#64] [General]
나중에 API 요청 많아지면 제대로 되고있는지 확인하기 좋을 것 같아서 관련 자료 남겨둡니다!

[사용법 적힌 공식문서](https://tanstack.com/query/v4/docs/framework/react/devtools)

<img width="890" alt="image" src="https://github.com/Part3-Team15/taskify/assets/24778465/9ab8f8cf-d0f2-48ae-9f4d-5c153b906514">

**Comment by wayandway** [comment]
친절한 설명 감사합니다!! ☺️
